RSSReader/View/FeedCell.m

```diff
@@ -105,11 +112,12 @@ - (UIView *)buttonView {
 }
 
 - (void)buttonTapped {
-    if ([self.isDescriptionShown isEqual:@1]) {
-        self.isDescriptionShown = @0;
-    }
-    else {
-        self.isDescriptionShown = @1;
+    switch ([self.isDescriptionShown intValue]) {
+        case SHOWN:
+            self.isDescriptionShown = [NSNumber numberWithInteger:HIDDEN];
```

Comment:
Use boxing NSNumber literal for enum value.

Reply:
fixed